src/test/java/org/takes/rs/RsWithCookieTest.java

```diff
@@ -104,6 +106,68 @@ public void rejectsInvalidName() {
      */
     @Test(expected = IllegalArgumentException.class)
     public void rejectsInvalidValue() {
-        new RsWithCookie(new RsEmpty(), "bar", "wo\"rks");
+        new RsWithCookie(new RsEmpty(), "cookiename", "wo\"rks");
+    }
+
+    /**
+     * Returns the joined cookie.
+     * @param name Cookie name
+     * @param value Cookie value
+     * @param path Cookie path
+     * @return Joined cookie
+     */
+    private String joiner(final String name, final String value,
```

Comment:
@dalifreire can't we have just 1 joiner method that takes multiple cookie strings ?